docs/interpreter_status.md

```diff
@@ -0,0 +1,216 @@
+test suite easy to maintain.
+<!-- markdownlint-enable line-length -->
+
+## Type Inference
+
+<!-- markdownlint-disable line-length -->
+The interpreter results are populated by indexing over the result type, so some
+development went into augmenting the type inference [implementation](https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/TypeInference.cpp)
+to position the interpreter to have more modular code. This allowed several
+complex ops like Convolution, Gather/Scatter, and ReduceWindow to be implemented
+nearly identical to how it is written in the spec.
+<!-- markdownlint-enable line-length -->
+
+## Special Cases
+
+### Miscellaneous
+
+This category has decomposable ops whose future is unclear at the moment. There
+are three specced ops in this category that the interpreter does not support at
+the moment:
+
+* `FftOp`
+* `RngOp`
+* `RngBitGeneratorOp`
+
+`FftOp` is categorized as Miscellaneous, but unlike other ops in this category,
+this op does not have an expander pass, and supporting this in StableHLO is a
+WIP.
+
+`RngOp` and `RngBitGeneratorOp` can be decomposed into MHLO ops, but the
+decomposition introduces a `XlaRngGetAndUpdateStateOp` which is an MHLO specific
+op. Supporting interpretation of these two ops is a WIP.
+
+<!-- markdownlint-disable line-length -->
+The tool to convert remaining ops in this category to StableHLO ops that the
+interpreter supports resides in [hlo_expand_main.cc](https://github.com/openxla/xla/blob/main/xla/tools/hlo_expand_main.cc).
+<!-- markdownlint-enable line-length -->
+
+### Not in HLO
+
+Apart from the specced ops, this category consists of 10 unspecced ops (as they
+plan to be moved out of StableHLO), some of which have existing passes in
+[mhlo](https://github.com/openxla/xla/tree/main/xla/mlir_hlo/mhlo/transforms) to
+convert them to StableHLO equivalent ops. There are three ops the interpreter
+does not support because there are no existing decompositions to StableHLO ops:
+
+* `compute_reshape_shape`
+* `cstr_reshapable`
+* `trace`
+
+`compute_reshape_shape` and `cstr_reshapable` ops are part of the ongoing
+Dynamism work, and they are planned to be removed from StableHLO (see
+[#1668](https://github.com/openxla/stablehlo/issues/1668)).
+
+`trace` op is private to XLA and there no no users in JAX, PyTorch or TensorFlow
+(see [#604](https://github.com/openxla/stablehlo/issues/604)).
+
+<!-- markdownlint-disable line-length -->
+The tool to convert remaining ops in this category to equivalent StableHLO ops
+that the interpreter supports resides in [mlir-hlo-opt.cc](https://github.com/openxla/xla/blob/main/xla/mlir_hlo/tools/mlir-hlo-opt/mlir-hlo-opt.cc).
+<!-- markdownlint-enable line-length -->
+
+### Quantization
+
+There are two specced ops in this category that the interpreter does not support
+at the moment. These ops are part of an ongoing Quantization work. Supporting
+interpretation of these two ops is a WIP:
+
+* `UniformDequantizeOp`
+* `UniformQuantizeOp`
```

Comment:
And uniform quant type. Do we have a ticket for this work?

I'd make this section simply say:
"Quantization spec'ing is still WIP, and interpreter support for quantized ops and types is tracked by ticket #????"

That will keep it from going out of date.